recipes-core/packagegroups/packagegroup-ni-extra.bb

```diff
@@ -148,7 +148,7 @@ RDEPENDS_${PN} += "\
 	python3-nose \
 	python3-numpy \
 	python3-pyiface \
-	python3 \
+	python3-modules \
```

Comment:
My understanding is that `python3-modules` is now providing the core python3 runtime and content. Since it's in the `-runmode` packagegroup, its contents will always end up in the core feed. So I think we can remove it from `-extra`.

Reply:
Done